modules/installation-creating-gcp-shared-vpc-ingress-firewall-rules.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
```

Comment:
This section is duplicate with Creating cluster-wide firewall rules for a shared VPC in GCP section. Please check.